tests/simulator/test_load_balancing.py

```diff
@@ -65,6 +65,20 @@ def density(x, y):
             + 1.0 / np.cosh((y - L * 0.7) / 0.5) ** 2
         )
 
+    def densityHigh(x, y):
+        assert cells == (100, 100)  # next lines are dependent
+        rho = y.copy()
+        rho[:] = 1e-10
```

Comment:
why not 0?

Reply:
I just had to be less than 1e-5 so it's arbitrary, could be 0